### PR TITLE
add a calibration program for the soloti

### PR DESCRIPTION
**Pull request by MaximilienNaveau**
[//]: # "Thanks for your contribution.  To make life of the reviewers easier,"
[//]: # "please give this pull request a meaningful title and provide the"
[//]: # "requested information below."

## Description

add the calibration routine for soloti

[//]: # "Description of what you did.  If it is more complex, consider to add a"
[//]: # "'Summary' section."

## How I Tested

[//]: # "Explain how you tested your changes"

tested with success on the real robot

## I fulfilled the following requirements

[//]: # "Please make sure you followed these steps before requesting a review."
[//]: # "Check the boxes in the list below, when done."

- [x] All new code is formatted according to our style guide (for C++ run clang-format, for Python, run flake8 and fix all warnings).
- [x] All new functions/classes are documented and existing documentation is updated according to changes.
- [x] No commented code from testing/debugging is kept (unless there is a good reason to keep it).

### CODE REVIEW

**Comment by MaximilienNaveau** [comment]
Hi @jviereck I just added you as reviewer to notify you of this. But eveything have been tested on the robot so I merge this fast to prevent delay on the robot.

File: src/programs/solo8ti_hardware_calibration.cpp [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    robot.calibrate(joint_index_to_zero);
+
+    long int count = 0;
+    while (!CTRL_C_DETECTED)
+    {
+        if (count % 200 == 0)
+        {
+            robot.acquire_sensors();
+            print_vector("Joint Positions", robot.get_joint_positions());
+        }
```
Comment (jviereck) [inline src/programs/solo8ti_hardware_calibration.cpp:30]: Note that `count` is not increased. So you are doing `count % 200 == 0` all the time.

Also, should there be a wait in the loop?

**Review by jviereck — APPROVED** [review]
Small question about loop body, but nothing major. Approving. Thanks for the PR.

**Comment by jviereck** [comment]
> Hi @jviereck I just added you as reviewer to notify you of this. But eveything have been tested on the robot so I merge this fast to prevent delay on the robot.

Sorry, saw this too late and did the review already.

**Pull request merged**